src/scripts.js

```diff
@@ -1 +1,48 @@
-console.log("Hello World");
\ No newline at end of file
+//Query Selectors
+let userInfo = document.querySelector(".user-info");
+let displayMessage = document.querySelector(".display-message");
```

Comment:
I would make this variable name a bit more descriptive. What message is it supposed to be displaying?

Reply:
We were thinking of doing displayWelcomeMessage but we were leaving it for now with thoughts of having the message change with different features we might add once we get through all the data calculations and functionality. But we can change it for now and go back later if we get there!